frontend/src/concepts/pipelines/const.ts

```diff
@@ -1,4 +1,4 @@
 export const PIPELINE_ROUTE_NAME = 'ds-pipeline-pipelines-definition';
-export const PIPELINE_DEFINITION_NAME = 'pipelines-definition';
+export const PIPELINE_DEFINITION_NAME = 'dsp-def';
```

Comment:
```suggestion
export const PIPELINE_DEFINITION_NAME = 'dspa';
```
This was my suggestion -- I don't think we need anything special here. Shorter the better -- but this is a minor nit -- since every resource will suffer the same restrictions of the 63 characters from the Route object